client/login/magic-login/index.jsx

```diff
@@ -477,6 +505,30 @@ class MagicLogin extends Component {
 			);
 		}
 
+		// "query?.redirect_to" is used to determine if Studio app users are creating a new account (vs. logging in)
```

Comment:
I do not follow this comment. From my testing, the `redirect_to` parameter is present for both logging in and account creation. 

What testing steps might I take to result in an outcome different that what is outlined in the PR description?

Reply:
@dcalhoun, to make sure I understand: are you seeing the `redirect_to` option when you choose "Email me a login link" from the main login page? That shouldn't be the case, the `redirect_to` option should only be used when choosing the options to create an account (it does also appear on the _main_ login page, but that isn't being targeted with the magic login code): 

https://github.com/Automattic/wp-calypso/assets/2998162/f397bc1f-1060-4432-b0a1-1cdbc7e04254

Reply:
Your screen recording and comment helped resolve my original confusion. I was seeking how I might arrive to a place where the `redirect_to` parameter was absent. As illustrated in my screenshot below, I now understand there are two URLs — `/log-in` and `/log-in/link` — and the `redirect_to` parameter comment on this line refers to its presence in the latter URL. 

<img width="1548" alt="log-in-url-parameters" src="https://github.com/Automattic/wp-calypso/assets/438664/c5b05872-76c6-4d17-8b91-0c88980570c7">


> are you seeing the `redirect_to` option when you choose "Email me a login link" from the main login page

No, I am not. I now believe it is functioning as expected.